python/ray/tune/suggest/hyperopt.py

```diff
@@ -52,21 +53,35 @@ class HyperOptSearch(SuggestionAlgorithm):
         >>>     'activation': 0, # The index of "relu"
         >>> }]
         >>> algo = HyperOptSearch(
-        >>>     space, max_concurrent=4, reward_attr="neg_mean_loss",
+        >>>     space, max_concurrent=4, metric="neg_mean_loss", mode="min",
         >>>     points_to_evaluate=current_best_params)
     """
 
     def __init__(self,
                  space,
                  max_concurrent=10,
-                 reward_attr="episode_reward_mean",
+                 reward_attr=None,
+                 metric="episode_reward_mean",
+                 mode="min",
                  points_to_evaluate=None,
                  **kwargs):
         assert hpo is not None, "HyperOpt must be installed!"
         from hyperopt.fmin import generate_trials_to_calculate
         assert type(max_concurrent) is int and max_concurrent > 0
+        assert mode in ["min", "max"], "mode must be 'min' or 'max'!"
+
+        if reward_attr is not None:
+            mode = "min"
+            metric = reward_attr
+            hyperopt_logger.warning("`reward_attr` will be depreciated!"
```

Comment:
Same comments as other files - also, you should use the standard logger instead of the hyperopt logger (do `logger = logging.getLogger(__name__)` under the imports, like nevergrad.py)